test/module/irohad/ordering/ordering_service_test.cpp

```diff
@@ -46,6 +48,28 @@ class OrderingServiceTest : public OrderingTest {
         grpc::CreateChannel(address, grpc::InsecureChannelCredentials()));
   }
 
+  void set(std::chrono::milliseconds d) {
```

Comment:
I am not so exiting about naming, what `d` means? I request to make names more convenient names.

Reply:
And how this variable used in method?

Reply:
1. just param that goes through, don't really matters. And also I don't really think a good idea to complicate var names w/o purpose
2. my fault, yeah. Should be in the 56 line instead of magic num